web/index.js

```diff
@@ -27,6 +27,12 @@ app.use(logger());
 app.use(cacheControlMiddleware);
 app.use(redirectMiddleware);
 app.use(iframeDestroyerMiddleware);
+
+// Check if the request url matches any assets inside of /dist
+app.use(serve(DIST_ROOT, {
+  maxage: 3600000, // set a cache time of one hour, helpful for mobile dev
```

Comment:
I set this to make mobile development over ngrok easier, so it can cache the rather large JS file that is compiled (and also cache the fonts)